Common/Models/WatchPredictedGlucose.swift

```diff
@@ -30,7 +30,7 @@ extension WatchPredictedGlucose: RawRepresentable {
     var rawValue: RawValue {
 
         return [
-            "v": values.map { Int16($0.quantity.doubleValue(for: .milligramsPerDeciliter)) },
+            "v": values.map { Int16($0.quantity.doubleValue(for: .milligramsPerDeciliter).clamped(to: Double(Int16.min)...Double(Int16.max))) },
```

Comment:
no idea. Could it be related to a recent algorithm change?

Reply:
I suppose it's possible with a crazy high ISF and a lot of boluses to actually trigger values this negative. I think it's probably ok to fix the conversion here, and if there is another problem with unexpected low forecast values, then this at least won't overshadow it. I'll create a new PR since the ticket was assigned to me, and I'm not sure what the other changes to the project are in this PR.

Reply:
Oh, I see what it's for now.  I'll just approve this one.